run.py

```diff
@@ -1,9 +1,24 @@
-from app import cli, create_app, db
+from app import cli, create_app, db, search_client, index
 from app.models import Category, Language, Resource, db
+import os
 
 app = create_app()
 cli.register(app, db)
 
+with app.app_context():
+    if os.environ.get('INDEX_CHECKED') == "FALSE":
+        query = Resource.query
+        indicies = search_client.list_indices()
+        for ind in indicies['items']:
+            if ind['name'] == 'resources_api':
```

Comment:
Are there two different indexes, one called `resources_api` and another called `search_resource_index`? I find this part rather confusing. Personally, I like the name `resources_api` for the index, and I only think we need one index. We should probably change the `.env` file to reflect this name if that's the one we're going with (and if it's the same index)

Reply:
I made it "search_resource_index" as a placeholder like the rest of the search related variables. 
I will fix this to get the indexes name from the environment variable.